builtin_metadata.json

```diff
@@ -342,6 +342,7 @@
       "v0.57.0",
       "v0.57.1",
       "v0.58.0",
+      "v0.59.0",
```

Comment:
I'm not sure what ramifications making this update now has .. but this change and adding `capabilities/v0.59.0.json` is necessary for the new tests to pass.

Does anyone know if this will mess anything up, and/or if there is an alternative?

Reply:
Looks like we're using published [git tags](https://github.com/open-policy-agent/opa/blob/89855df570ef38cf5d87e5542a02d086be86af83/docs/website/scripts/load-docs.sh#L22C3-L22C3) for building the `releases.yaml` file used for iterating versions in the docs. So this might be safe after all, at least when it concerns docs ..

Reply:
Do you have an idea if this has any impact when we cut the actual v0.59.0 release? I would imagine the release scripts would just overwrite this but I'm not sure. Als it's a bit weird we have published v0.59.0 artifacts w/o an actual release but that's ok I guess.

Reply:
Looking at the web deployment preview of [#6346](https://github.com/open-policy-agent/opa/pull/6346), it doesn't look like this will break the documentation.

Reply:
Looking at the web deployment preview of [#6346](https://github.com/open-policy-agent/opa/pull/6346), it doesn't look like this will break the documentation.

Reply:
Running the `release-patch` make target on this branch also has no unexpected ill effects. So I think we're good to go here :) .